.github/workflows/alpha-integration-tests.yaml

```diff
@@ -1,10 +1,14 @@
-name: DataRepo Alpha Integration Tests
+name: DataRepo Alpha Tests and Prod GCR Promotion
 env:
   K8_CLUSTER: jade-master-us-central1
   GOOGLE_APPLICATION_CREDENTIALS: /tmp/alpha-test-runner.json
   GOOGLE_CLOUD_PROJECT: terra-datarepo-alpha
   GOOGLE_CLOUD_DATA_PROJECT: terra-datarepo-alpha-data
   TDR_LOG_APPENDER: Console-Standard
+  GCR_DEV_URL: gcr.io/broad-jade-dev/
+  GCR_PROD_URL: gcr.io/terra-datarepo-production/
```

Comment:
I recommend removing the trailing slash here and adding it in the concatenation step instead. It isn't an issue in this particular case, but in the general case this will ensure that the URL / file path is well-constructed regardless of whether there are trailing slashes in the input or not (I would still remove it to avoid confusion). If, on the other hand the slashes were missing in both the input and the concatenation, this would produce an invalid URL / file path. This is because the URL / POSIX standard specifies that the path `a///b` is considered valid and servers / bash typically treats `a///b` as `a/b`.